src/_guides/language/language-tour.md

```diff
@@ -252,6 +259,10 @@ the first time it's used.
 <div class="alert alert-info" markdown="1">
 **Note:**
 Instance variables can be `final` but not `const`.
+Final instance variables must be initialized before
+the constructor body starts —
+at the variable declaration, in a constructor argument,
```

Comment:
"in a constructor argument" reads weirdly to me (but it just might be me).
Maybe "by a constructor argument"? (Or, better, "by a constructor parameter" - we try to use "parameter" about the function's declared parameters, and "argument" about the value being passed at the call site).